coordinator/quote/ertvalidator/ertvalidator.go

```diff
@@ -13,7 +13,9 @@ import (
 	"encoding/hex"
 	"fmt"
```

Comment:
```suggestion
	"encoding/hex"
	"errors"
	"fmt"
```
missing import of `errors` package

Reply:
Thank you, sorry for that, fixed in the last force push.